src/utils.rs

```diff
@@ -1,18 +1,15 @@
-use bcrypt_pbkdf::bcrypt_pbkdf;
+use crate::viewing_key::VIEWING_KEY_SIZE;
+use sha2::{Digest, Sha256};
+use std::convert::TryInto;
 use subtle::ConstantTimeEq;
 
-// 5 rounds == ~300M gas (doesn't work with query) - creation/validation takes ~1.5s
-// 2 rounds == ~120M gas (works with query) - creation/validation takes ~1s
-const DEFAULT_COST: u32 = 2;
-const OUTPUT_SIZE: usize = 24;
-const SALT: &[u8] = b"bestspiceintheEU";
-
 pub fn ct_slice_compare(s1: &[u8], s2: &[u8]) -> bool {
     bool::from(s1.ct_eq(s2))
 }
 
-pub fn create_hashed_password(s1: &str) -> [u8; OUTPUT_SIZE] {
-    let mut output = [0u8; OUTPUT_SIZE];
-    let _ = bcrypt_pbkdf(s1, SALT, DEFAULT_COST, &mut output);
-    output
+pub fn create_hashed_password(s1: &str) -> [u8; VIEWING_KEY_SIZE] {
+    Sha256::digest(s1.as_bytes())
```

Comment:
this can be more flexible if you say that the argument to `create_hashed_password` is a `&[u8]`, but it would make the call to it a little more complicated.

Reply:
What do you mean more flexible? We always get the password from the user, which is always a String so I think it makes sense? (I mean, we don't really need a generic `sha256()` func since it's a one-liner anyway)

Reply:
oh, if it's always a string then yeah, nothing to change